lief/README.md

```diff
@@ -0,0 +1,19 @@
+# lief
+
+[stoQ](https://stoq-framework.readthedocs.io/en/latest/index.html) plugin that parses and abstracts PE, ELF and MachO executable files using LIEF and produces metadata about the payload.
```

Comment:
Do you mind adding an inline link to `LIEF` so potential users can check it out and get a better understanding of what it offers?